variables.yaml

```diff
@@ -132,32 +132,150 @@ netcdf_variables:
 
+  backscatter:
+    source:       FLBBPC_BB_700_SCALED
+    long_name:    700 nm wavelength backscatter
+    standard_name: 700_nm_wavelength_backscatter_in_sea_water
+    units:        m-1
+    coordinates:  time depth latitude longitude
+    instrument:    instrument_scatterometer
+    valid_min:    "0."
+    valid_max:    "3."
+    observation_type: "calculated"
+    sensitivity:      "0.002"
 
-# Oxygen
+  chlorophyll2:
+    source:       FLBBCD_CHL_SCALED
+    long_name:    chlorophyll
+    standard_name: concentration_of_chlorophyll_in_sea_water
+    units:        mg m-3
+    coordinates:  time depth latitude longitude
+    instrument:    instrument_flourometer
+    valid_min:    "0."
+    valid_max:    "50."
+    observation_type: "calculated"
+    sensitivity:      "0.025"
+
+  cdom:
+    source:       FLBBCD_CDOM_SCALED
+    long_name:    colored dissolved organic matter
+    standard_name: concentration_of_colored_dissolved_organic_matter_in_sea_water
+    units:        ppb
+    coordinates:  time depth latitude longitude
+    instrument:    instrument_flourometer
+    valid_min:    "0."
+    valid_max:    "375."
+    observation_type: "calculated"
+    sensitivity:      "0.28"
+
+  backscatter:
+    source:       FLBBCD_BB_700_SCALED
+    long_name:    700 nm wavelength backscatter
+    standard_name: 700_nm_wavelength_backscatter_in_sea_water
+    units:        m-1
+    coordinates:  time depth latitude longitude
+    instrument:    instrument_scatterometer
+    valid_min:    "0."
+    valid_max:    "3."
+    observation_type: "calculated"
+    sensitivity:      "0.002"
+
+
+# oxygen
   oxygen_concentration:
     source:       AROD_FT_DO
     long_name:    oxygen concentration
     standard_name: mole_concentration_of_dissolved_molecular_oxygen_in_sea_water
     units:        umol l-1
     coordinates:   time depth latitude longitude
+    instrument:    instrument_phosphorescence
+    valid_min:    "0."
+    valid_max:    "425."
+    observation_type: "calculated"
+    accuracy:      "2% or 2.0"
+    resolution:      "0.01"
 
   temperature_oxygen:
     source:  AROD_FT_TEMP
     long_name:    oxygen sensor temperature
     standard_name: temperature_of_sensor_for_oxygen_in_sea_water
-    units:        Celsius
+    units:        degree Celsius
     coordinates:   time depth latitude longitude
+    instrument:    instrument_thermistor
+    valid_min:    "-3."
+    valid_max:    "45."
+    observation_type: "measured"
+    accuracy:      "0.01"
+    resolution:      "0.001"
 
+  oxygen_concentration2:
+    source:       LEGATO_CODA_DO
+    long_name:    oxygen concentration
+    standard_name: mole_concentration_of_dissolved_molecular_oxygen_in_sea_water
+    units:        umol l-1
+    coordinates:   time depth latitude longitude
+    instrument:    instrument_optode
+    valid_min:    "0."
+    valid_max:    "500."
+    observation_type: "calculated"
+    accuracy:      "5% or 8.0"
+    resolution:      "<1"
```

Comment:
Leave out the < sign, just a number is good. Better to have the largest resolution value the manufacturer quotes